lib/cli/utils.ts

```diff
@@ -7,17 +7,25 @@ const assert = require('assert');
 const glob = require('glob');
 const resolve = require('resolve-pkg');
 
+
+interface ILSCSettings {
```

Comment:
- interface names should **not** start with `I` letter
- abbreviations in names should be treated as words `ABCSettings` -> `AbcSettings`